### ISSUE TITLE
Need new maintainer for https://github.com/conda-forge/saspy-feedstock

### ISSUE THREAD

**Issue by tomweber-sas** [conda-forge/conda-forge.github.io#1441]
Trying to fix the conda forge process for building [saspy](https://github.com/conda-forge/saspy-feedstock) 
This was being done by @danielfrg, but apparently not anymore. And, now it's broke and hasn't built a release for a while.
I'm not familiar with this whole process, but I'm trying to get this issue fixed. Seems it need a maintainer. I can be that if necessary. Is it Anaconda employee's that maintain these things? Any person who offers? Again, I'm unfamiliar with any of this. So thanks for any help you can provide.
Tom

**Comment by beckermr** [comment]
Anyone who offers can do it. We work with anaconda inc but are a separate entity. Please make a PR to add yourself to the feedstock and a member of core can merge it. Recipe maintainers are listed in the extra section of the meta.yaml in the recipe directory of the feedstock repo.

**Comment by tomweber-sas** [comment]
Hey, cool, thanks! I'll go do that now. I'll close this after it get's fixed. I had added a number of @conda-forge-admin comments and in issues too, but nothing's ever been responded to. Should I add someone to that request so they see it @ core or something?
Thanks!
Tom

**Comment by beckermr** [comment]
the admin is a bot, you want core.

**Comment by tomweber-sas** [comment]
Thanks guys! Am I now the 'maintainer'? Also, I have 3 versions that never built. How do I get those rebuilt? Just do that bot-rerun thing now?

**Comment by tomweber-sas** [comment]
ah, email

**Comment by beckermr** [comment]
yep. accept the invite. Then merge the latest version PR if you don't need the previous ones.

**Comment by jakirkham** [comment]
> yep. accept the invite. Then merge the latest version PR if you don't need the previous ones.

xref: https://github.com/conda-forge/saspy-feedstock/commit/0fa0500b61762ea5b255e8c56ad0a351c1079507#comments

**Comment by tomweber-sas** [comment]
Thanks, did that. Now I'll work through each in order, so each version is available. I was able to merge the two previous versions, and they now show up at https://anaconda.org/conda-forge/saspy/files, but the current version 3.7.1 PR (https://github.com/conda-forge/saspy-feedstock/pull/57) was closed and I can't re-open it to get it to merge. How do I re-trigger the process that creates these, which I guess happens when I build a new version on Pypi, or is it on my saspy repo? I'd like to get the current version on conda too.
Thanks again, and sorry for the newbie questions; this has always been automatic so I've never had to look at it before.

**Comment by jakirkham** [comment]
The bot will add a new PR ( https://github.com/conda-forge/saspy-feedstock/pull/57#issuecomment-840810004 ). Though the bot runs on a cron job. So it may be a little bit before the PR comes. Could either wait for that bot PR or make one manually

**Comment by tomweber-sas** [comment]
ok, I'm shutting down for the day. I'll wait for it to re-trigger and check on it later.
Thanks again! hopefully me last question for a while!
Appreciate all your patience with me.
Thanks,
Tom

**Comment by jakirkham** [comment]
Thanks for helping out Tom 🙂

**Comment by tomweber-sas** [comment]
Just to follow up, the bot rocks :) It picked up that 3.7.1 was still out there and hadn't been integrated, so it did a fresh PR and I just merged it. Nice!  Out of curiosity, what's it keying off of? Releases that are published on my repo (https://github.com/sassoftware/saspy/releases), or what exactly? Can't be the release.py file in the main branch (hope not). I used to think it was PyPI, but after today, it seems like it must be off my repo, really. I'll get up to speed on this and read the doc I got links to as soon as I get a chance. I think I'm good for a while now. Just wanted to get these release available to my customers who like to do conda installs instead of pip installs. So, I'm all caught up now!
Again, that's for all of your help, I really appreciate it. 
Tom

**Comment by jakirkham** [comment]
It's based off of [the source URL in the recipe]( https://github.com/conda-forge/saspy-feedstock/blob/2660c1c44c0be25d16b1d77703758a01c7dc575e/recipe/meta.yaml#L11 ). In this case it appears to be PyPI, but it could use GitHub or other URLs

**Comment by tomweber-sas** [comment]
plain as day :)  ... thanks!

**Comment by tomweber-sas** [comment]
Hey y'all, it's time to transition this to others. I opened an issue on my repo yesterday, but I kinda think it's only me that will see it. I found this from years ago when I took on maintaining this repo (since I created saspy and have been supporting it since then), as it's just merging in PR's to build new releases the Bot sees after I push to PyPI. I don't know if anyone with ability will see https://github.com/conda-forge/saspy-feedstock/issues/125. but I don't seem to have permission to add others.
Thanks in advance!
Tom
